addons/bft/functions/fnc_syncedArrayPushback.sqf

```diff
@@ -16,6 +16,6 @@
 
 params ["_varName", "_data"];
 
-if (typeName _data == "ARRAY" && {(count _data == 0)}) exitwith {};
+if (typeName _data == "ARRAY" && {_data isEqualTo []}) exitwith {};
```

Comment:
isEqualType

Also the check is redundant. the isEqualTo already checks that it's an array.